src/main/java/com/kneelawk/extramodintegrations/tconstruct/Util.java

```diff
@@ -1,10 +1,25 @@
 package com.kneelawk.extramodintegrations.tconstruct;
 
+import java.util.ArrayList;
+import java.util.List;
 import dev.emi.emi.api.stack.EmiStack;
 import io.github.fabricators_of_create.porting_lib.fluids.FluidStack;
+import net.minecraft.text.Text;
+import net.minecraft.util.Formatting;
+import slimeknights.mantle.fluid.tooltip.FluidTooltipHandler;
 
 public class Util {
     public static EmiStack convertFluid(FluidStack fluidStack) {
         return EmiStack.of(fluidStack.getFluid(), fluidStack.getTag(), fluidStack.getAmount());
     }
+
+    public static List<Text> getFluidTiCTooltip(FluidStack fluidStack) {
+        List<Text> tooltip = new ArrayList<Text>();
+        if (FluidTooltipHandler.appendMaterialNoShift(fluidStack.getFluid(), fluidStack.getAmount(), tooltip)) {
+            return tooltip.stream().map((Text text) -> {
+                return (Text) Text.literal(text.getString()).formatted(Formatting.BLUE);
```

Comment:
is it necessary to convert the Text to a String here? is it necessary to modify it at all? the REI plugin does not apply this styling.

Reply:
@Abbie5 IMO the styling is necessary to differentiate between the liquid amount shown by EMI and the "human readable amount" shown by tinkers.  The output isn't always terse like "1 ingot". It can be mix or block, ingot, gem, shard and leftover mB. So without different color its very confusing.